Chapter1/1_Is Unique/IsUnique.py

```diff
@@ -18,19 +18,17 @@ def unique(string):
     return True
 
 
-class Test(unittest.TestCase):
-    dataT = [('abcd'), ('s4fad'), ('')]
-    dataF = [('23ds2'), ('hb 627jh=j ()')]
+class IsUniqueTest(unittest.TestCase):
 
-    def test_unique(self):
-        # true check
-        for test_string in self.dataT:
-            actual = unique(test_string)
-            self.assertTrue(actual)
-        # false check
-        for test_string in self.dataF:
-            actual = unique(test_string)
-            self.assertFalse(actual)
+    def test_true_check(self):
+        dataT = [('abcd'), ('s4fad'), ('')]
```

Comment:
Hi! Probably need to update this PR, as more test cases have been added upstream, and formatting changed a bit. :-)